tests/Entity/TransformerTest.php

```diff
@@ -290,6 +281,9 @@ public function blockProvider(): iterable
      */
     private function getTestContent(\DateTimeInterface $datetime): array
     {
+        /**
+         * @var numeric-string $dateTimeString
```

Comment:
I think the PHPStan error is because they are not return type provider (same than it exists for psalm) I'll open an issue.